pkg/datapath/cells.go

```diff
@@ -52,6 +53,10 @@ var Cell = cell.Module(
 	// The monitor agent, which multicasts cilium and agent events to its subscribers.
 	monitorAgent.Cell,
 
+	// The modules manager to search and load kernel modules.
+	// Must appear before the datapath so that its start hook is registered before the iptables one.
```

Comment:
This comment about ordering is not true. Since `newDatapath` depends on the modules manager it will be started before iptables. Also since `modules.Cell` does not have an invoke, it doesn't matter where it is here since the order only matters for invokes.

Reply:
Got confused with `Invoke` and `OnStart`, I'm gonna fix this :+1:

Reply:
Thanks! :smiley: